package.json

```diff
@@ -730,6 +730,11 @@
           "description": "%java.debugger.configuration.jdwp.requestTimeout.description%",
           "default": 3000,
           "minimum": 100
+        },
+        "java.debug.settings.vmArgs": {
```

Comment:
Do you think if it is necessary to set it as `machine` scope?

Reply:
Seems not, because `vmArgs` is not auto executed by the debugger. It's like `vmArgs` in launch.json, need user to click debug to consume it.

Reply:
I see, that's reasonable.